services/gitea/gitea-helper.js

```diff
@@ -1,5 +1,6 @@
 const documentation = `
-Note that the gitea_url parameter is required because there is canonical hosted gitea service provided by Gitea.
+Note that only internet-accessible Gitea/Forgejo based instances are supported, for example https://gitea.com, https://codeberg.org.
+For self-hosted instances, the gitea_url parameter is required.
```

Comment:
Can we reword this to make it more clear that `gitea_url` defaults to https://gitea.com when not supplied.

How about something like:

```
By default this badge looks for repositories on [gitea.com](https://gitea.com).
To specify another instance like [codeberg](https://codeberg.org/), [forgejo](https://forgejo.org/) or a self-hosted instance, use the `gitea_url` query param.
```

?

Reply:
Agreed, this is much better verbiage.